integration_tests/src/main/python/conftest.py

```diff
@@ -87,6 +87,9 @@ def is_emr_runtime():
 def is_dataproc_runtime():
     return runtime_env() == "dataproc"
 
+def is_dataproc_serverless_runtime():
+    return runtime_env() == "dataproc_serverless"
```

Comment:
Probably we also need to update the [README](https://github.com/NVIDIA/spark-rapids/tree/branch-24.08/integration_tests#runtime-environment) about this new value